### PR TITLE
A simple build wrapper example with Timestamper

### PR DESCRIPTION
**Pull request by abayer**
@reviewbybees

### CODE REVIEW

File: timestamper-wrapper/timestamperWrapper.groovy [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// This shows a simple build wrapper example, using the Timestamper plugin.
+node {
+    // This is the current syntax for invoking a build wrapper, naming the class.
+    wrap([$class: 'TimestamperBuildWrapper']) {
```
Comment (rtyler) [inline timestamper-wrapper/timestamperWrapper.groovy]: Just curious, from a Groovy perspective can you implement this as an inline function like one would do normally in Groovy, e.g.

```groovy
void timestamper(Closure nested) {
  wrap([$class: 'TimestamperBuildWrapper']) {
    nested.call()
  }
}

node {
  timestamper {
    /* etc */
  }
}
```

**Comment by rtyler** [comment]
:bee:

File: timestamper-wrapper/timestamperWrapper.groovy (hunk 1 above)
Comment (abayer) [inline timestamper-wrapper/timestamperWrapper.groovy]: Think so, yeah.

**Comment by amuniz** [comment]
:bee:

File: timestamper-wrapper/timestamperWrapper.groovy (hunk 1 above)
Comment (jglick) [inline timestamper-wrapper/timestamperWrapper.groovy]: Yes. BTW more simply

```groovy
nested()
```

**Pull request merged**